packages/compiler-dom/src/decodeHtmlBrowser.ts

```diff
@@ -1,8 +1,16 @@
 /* eslint-disable no-restricted-globals */
 
+const encodeRE = /<|>/g
+const encodeMap: Record<string, string> = {
+  '<': '&lt;',
+  '>': '&gt;'
+}
+
 let decoder: HTMLDivElement
 
 export function decodeHtmlBrowser(raw: string): string {
-  ;(decoder || (decoder = document.createElement('div'))).innerHTML = raw
+  ;(
+    decoder || (decoder = document.createElement('div'))
+  ).innerHTML = raw.replace(encodeRE, (_, p1) => encodeMap[p1])
```

Comment:
When I tried running this code it didn't seem to work. I think `(_, p1)` should be just `(p1)`.